docs/UserGuide.md

```diff
@@ -155,10 +157,24 @@ This is one of the most used command type in CheatLogs. They allow you to manage
 ___
 
 <a id="add-command"></a>
-### 4.1.1. Adding a cheatsheet: `/add` <font size="5"> [:arrow_up_small:](#table-of-contents)</font>
+#### 4.1.1. Adding a cheatsheet: `/add` <font size="5"> [:arrow_up_small:](#table-of-contents)</font> 
+You can easily use the **/add** command to add your own cheat sheets to the list by using 2 different methods **easy** and **hard** mode.
+#### 4.1.1.1 Adding a cheatsheet using the easy command
+The first method that you can use to add cheatsheet is the easy command. Just type `/add` without adding any flag or description. 
+Format: `add`
 
->Format: `/add /n CHEATSHEET_NAME /s SUBJECT` <br>
->Flag optionality: `/n`(required), `/s` (optional)
+You will be prompted to fill in the `NAME` and the `SUBJECT` of your cheatsheet.
+
+Shortly after, a window will pop up to show the editing window.
+Enter the details of your cheatsheet there, then click `Actions` then `Save`
+![image](https://i.ibb.co/n01kQ4Y/carbon.png)
+Congratulations! You have added your first cheatsheet into CheatLogs.
+
+#### 4.1.1.2 Adding a cheatsheet using the advanced mode
+For the more advanced users, you can also add cheatsheets using the advanced command
+
+Format: `/add /n <CHEATSHEET_NAME> /s <SUBJECT>` <br>
+Flag optionality: `/n` (required)`/s` (optional)
```

Comment:
could perhaps indent this format and flag lines to be consistent with other commands